### PR TITLE
Use activerecord "human.attribute_name"

### PR DESCRIPTION
**Pull request by neomilium**
Affected keywords were missing. Instead of create duplicates I use already translated activerecord attributes.

### CODE REVIEW

**Comment by neomilium** [comment]
@smortex, @manno  I'm not sure if its a good practice to address this problem. Comments are welcome.

File: app/views/public/schedule/_events.html.haml [hunk 1]
```diff
@@ -7,8 +7,8 @@
       %thead
         %tr
           %th &nbsp;
-          %th= t '.title'
-          %th= t '.speakers'
+          %th= t 'activerecord.attributes.event.title'
```
Comment (smortex) [inline app/views/public/schedule/_events.html.haml]: I would have used [human_attribute_name](http://api.rubyonrails.org/classes/ActiveModel/Translation.html#method-i-human_attribute_name):

```ruby
Event.human_attribute_name('title')`
```

Comment (smortex) [reply app/views/public/schedule/_events.html.haml]: That way, no need to be aware of how things are organized in i18n files to understand what the code is trying to do :stuck_out_tongue_winking_eye:

**Comment by neomilium** [comment]
@smortex  Thanks! I updated my commit and update existing translation based on 'activerecord.attributes' using:

```
find app/views/ -exec sed -e "s/\(.*\)t.'activerecord\.attributes\.\([^.]*\)\.\(.*\)')*/\1\u\2.human_attribute_name('\3')/" -i '{}' \;
```

File: app/views/conference_users/_table.html.haml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 %table.zebra-striped
   %thead
     %th.first
-    %th= t('activerecord.attributes.user.email')
-    %th= t('activerecord.attributes.conference.acronym')
-    %th= t('activerecord.attributes.conference_user.role')
+    %th= User.human_attribute_name('email')
+    %th= Conference.human_attribute_name('acronym')
+    %th= Conference_user.human_attribute_name('role')
```
Comment (smortex) [inline app/views/conference_users/_table.html.haml]: `Conference_user` is not a valid class name in respect to Ruby conventions.  Typo?

Comment (neomilium) [reply app/views/conference_users/_table.html.haml]: Commit fixed. Thanks1

**Comment by neomilium** [comment]
@manno This PR is now complete and can be merged.

**Pull request merged**